gremlin-dotnet/src/Gremlin.Net/Process/Utils.cs

```diff
@@ -32,6 +33,8 @@ namespace Gremlin.Net.Process
     /// </summary>
     internal static class Utils
     {
+        public static readonly string UserAgent = GenerateUserAgent();
```

Comment:
A public field is usually discouraged in C#. It isn't a big problem here as this class is `internal`, but I would still suggest that only make properties and methods `public`. My suggestion is to turn this into a simple property with a backing field:
```suggestion
        public static string UserAgent => _userAgent ??= GenerateUserAgent();
        private static string _userAgent;
```